x/staking/src/keeper/redelegation.rs

```diff
@@ -348,7 +343,7 @@ impl<
         };
 
         let storage = InfallibleContextMut::infallible_store_mut(ctx, &self.store_key);
-        let mut store = storage.prefix_store_mut(UNBONDING_QUEUE_KEY);
+        let mut store = storage.prefix_store_mut(REDELEGATION_QUEUE_KEY);
```

Comment:
Did you only rename key or changed it value? If you did, then check state hash of main and your branch

Reply:
It was wrong key there.

I'll check.

Reply:
It's okay.